.github/workflows/lint.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - uses: crate-ci/typos@v1.19.0 # selfup { "regex": "\\d\\.\\d+\\.\\d+", "script": "typos --version | cut -d ' ' -f 2" }
+      - uses: crate-ci/typos@v1.20.1 # selfup { "regex": "\\d\\.\\d+\\.\\d+", "script": "typos --version | cut -d ' ' -f 2" }
```

Comment:
Ah... not yet applied https://github.com/NixOS/nixpkgs/pull/301147

Reply:
False positive detections, see https://github.com/crate-ci/typos/pull/973

Reply:
https://github.com/kachick/dotfiles/actions/runs/8544089356/job/23409353597?pr=546

```
2024/04/03 19:01:11 typos . .github .vscode
error: `guid` should be `guide`
[cargo-make] ERROR - Error while executing command, exit code: 1
[cargo-make] WARN - Build Failed.
  --> ./config/windows/WindowsTerminal/settings.json:52:10
   |
52 |         "guid": "{61c54bbd-c2c6-5271-96e7-009a87ff44bf}",
   |          ^^^^
   |
error: `guid` should be `guide`
  --> ./config/windows/WindowsTerminal/settings.json:58:10
   |
58 |         "guid": "{0caa0dad-35be-5f56-a8ff-afceeeaa6101}",
   |          ^^^^
   |
error: `guid` should be `guide`
  --> ./config/windows/WindowsTerminal/settings.json:63:10
   |
63 |         "guid": "{b453ae62-4e3d-5e58-b989-0a998ec441b8}",
   |          ^^^^
   |
error: `guid` should be `guide`
  --> ./config/windows/WindowsTerminal/settings.json:70:10
   |
70 |         "guid": "{574e775e-4f2a-5b96-ac1e-a2962a402336}",
   |          ^^^^
   |
error: `guid` should be `guide`
  --> ./config/windows/WindowsTerminal/settings.json:76:10
   |
76 |         "guid": "{5[185](https://github.com/kachick/dotfiles/actions/runs/8544089356/job/23409353597?pr=546#step:7:186)5cb2-8cce-5362-8f54-464b92b32386}",
   |          ^^^^
   |
error: `guid` should be `guide`
  --> ./config/windows/WindowsTerminal/settings.json:85:10
   |
85 |         "guid": "{2c4de342-38b7-51cf-b940-2309a097f518}",
   |          ^^^^
   |
```

Reply:
https://github.com/kachick/dotfiles/pull/546/commits/fccdc0245b49c7b8434faf496e49abf7ace9e4b1